src/plan_bot.py

```diff
@@ -234,18 +243,22 @@ def process_post(
 
     post_text, options = process_flags(get_trigger_line(post.text))
 
-    match_info = (
-        RuleStrategy.request_help(plans, post_text, post=post)
-        or RuleStrategy.request_plan_list(plans, post_text, post=post)
-        or RuleStrategy.match_display_title(plans, post_text, post=post)
-        or matching_strategy(plans, post_text, post=post)
-    )
+    if options.get("why_warren"):
```

Comment:
Hmmm, this is matching logic leaking into the `process_post` function. Can this be taken care of by a RuleStrategy?

Perhaps you'll want to have a RuleStrategy that checks options against all known verbatims

Reply:
Feel free to extend the signature for `RuleStategy` methods to allow an `options` dict to be passed in. That should allow you to move more logic over there.